pallets/schemas/src/migrations.rs

```diff
@@ -0,0 +1,23 @@
+//! Migrations for the schemas pallet.
+//! Following migrations are required:
+//! - Adding settings to the Schema struct
+//! Note: Post migration, this file should be deleted.
+use crate::{types::Schema, *};
+use common_primitives::schema::SchemaSettings;
+use frame_support::pallet_prelude::Weight;
+
+pub fn migrate_schemas_with_additional_settings<T: Config>() -> Weight {
+	let mut weight: Weight = Weight::zero();
+	//translate_values of Schemas storage add settings: SchemaSettings::all_disabled()
+	//to all schemas
+	<Schemas<T>>::translate_values(|old_schema: Schema<T::SchemaModelMaxBytesBoundedVecLimit>| {
```

Comment:
I don't see us using different Schema types for old and new ones. How does substrate know how to decode old values? I'm not sure if this will work

Reply:
I also like enwrapping the migration inside a mod version 

```rust
mod v1{
    // migration logic
}
```
example https://github.com/paritytech/substrate/blob/master/frame/contracts/src/migration.rs#L87

Reply:
In general through any language or db i expect decoders to just populate fields that exists, hoping its the same here as I expect this to working similar to going through 
https://github.com/paritytech/polkadot/pull/4545

Reply:
I also like the ides of modulating it, will do after these arduous upgrade testing

Reply:
but even in that example there is 2 versions for **SessionInfo**
```

<Pallet<T> as Store>::Sessions::translate_values(|old: primitives::**v1**::SessionInfo| {
		vs += 1;
		Some(primitives::**v2**::SessionInfo::from(old))
```